api.yaml

```diff
@@ -165,7 +165,7 @@ info:
   contact:
     name: "Tiqets API Support"
     email: apisupport@tiqets.com
-    version: 2.0.0
+    version: 2.1.0
```

Comment:
My previous comment didn't appear 😩 We need to move the version to the same level as `contact`  

https://swagger.io/specification/#info-object